README.md

```diff
@@ -1,12 +1,64 @@
-# ase
+# Grout Schema Editor
 
-This project is generated with [yo angular generator](https://github.com/yeoman/generator-angular)
-version 0.11.1.
+A simple, static admin portal providing an interface to a Grout database.
 
-## Build & development
+## Requirements
 
-Run `grunt` for building and `grunt serve` for preview.
+The Grout Schema Editor is containerized with Docker for ease of use. In order
+to develop with Docker, you need the following dependencies:
+
+- [Docker CE Engine](https://docs.docker.com/install/) >= 1.13.0 (must be
+  compatible with [Docker Compose file v3
+  syntax](https://docs.docker.com/compose/compose-file/#compose-and-docker-compatibility-matrix))
+- [Docker Compose](https://docs.docker.com/compose/install/)
+
+## Use the schema editor in a project 
+
+To incorporate the Grout Schema Editor into a project, clone it into your project
+repo:
+
+```
+# Run this command in your project directory.
+git clone git@github.com:azavea/grout-server.git
+```
+
+You can also manage the dependency using [git
+subtree](https://www.atlassian.com/blog/git/alternatives-to-git-submodule-git-subtree)
+if you plan to contribute your changes back upstream:
+
+```bash
+# Add the grout-schema-editor repo as a remote to your project directory.
+git remote add -f grout-schema-editor git@github.com:azavea/grout-schema-editor.git
+
+# Pull in grout-schema-editor as a subtree in your project.
+git subtree add --prefix grout-schema-editor grout-schema-editor master
+```
+
+Next, edit the app config file to match the requirements of your project. The
```

Comment:
Generally for configuration files like this, we put the version that lives in git in a top-level `example` directory (or something like that, so maybe `example/config.js` here), and then instruct the user in the README to copy the example file to where the app needs it and then edit it. With that, we can git-ignore the file used by the app (`app/scripts/config.js`, here), so that the user edits don't show up as a diff.